app/services/accredited_provider_api.rb

```diff
@@ -0,0 +1,20 @@
+class AccreditedProviderApi
+  include ServicePattern
+
+  def call
+    Rails
+      .cache
+      .fetch("accredited_provider_api", expires_in: 24.hours) do
+        response = HTTParty.get(provider_url)
+        response = JSON.parse(response.to_s)
+        response["data"]
+      end
+  end
+
+  private
+
+  def provider_url
+    year = Time.current.next_year.year
+    "https://www.publish-teacher-training-courses.service.gov.uk/api/public/v1/recruitment_cycles/#{year}/providers?filter[is_accredited_body]=true"
```

Comment:
Can the base URL be an ENV variable?

Reply:
resolved in [abdfae3](https://github.com/DFE-Digital/itt-mentor-services/pull/48/commits/abdfae3e4ab05b2c4cf0a45a0c634fec1fbf6545)